hw19

irbis45 opened the pull request:
планирование задач

egerasimchuk commented on database/migrations/2020_01_01_181516_create_orders_table.php:
```diff
@@ -0,0 +1,41 @@
+    {
+        Schema::create('orders', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('order_number', 50);
```
Лучше просто number таблица orders, и скорее всего поле должно быть уникальным

egerasimchuk commented on database/migrations/2020_01_01_181516_create_orders_table.php:
```diff
@@ -0,0 +1,41 @@
+            $table->bigInteger('address_id')->unsigned();
+            $table->bigInteger('delivery_id')->unsigned();
+            $table->text('user_message')->nullable();
+            $table->tinyInteger('order_status_id')->unsigned()->nullable();
```
Тоже модно просто status_id

irbis45 committed:
hw19

# планирование задач

irbis45 committed:
hw19

# поправил docker-compose.yml. Для scheduler имя контейнера теперь задается автоматически
